pfioh/pfioh.py

```diff
@@ -1046,17 +1047,10 @@ def do_POST_withCompression(self, **kwargs):
         #Decoding 
         d_ret['write']   = {}
 
-        try:
-            data = fileContent.decode()
-            d_ret['write']['decode'] = True
-        except Exception as err:
-            d_ret['write']['decode'] = False
-            data = fileContent
-
         b_zip = b_unpack and (d_compress['archive'] == 'zip')
 
         d_ret= self.storeData(file_name= str_localFile, client_path = str_fileName,
-                              file_content= data, Path= str_unpackPath, is_zip=b_zip,d_ret=d_ret)
+                              input_stream=inputStream, Path= str_unpackPath, is_zip=b_zip,d_ret=d_ret)
```

Comment:
The input stream is never converted from bytes, since it never needs to be read

Reply:
space before and after `=`